ptychonn/pospred/message_logger.py

```diff
@@ -0,0 +1,7 @@
+import logging
+logging.getLogger(__name__).setLevel(logging.INFO)
+
+
+logger = logging
+logger.basicConfig(format='[%(asctime)s] %(message)s', level=logging.INFO)
+
```

Comment:
As mentioned, configuring the logger is a task for the end user, not for a library.